process/factory/shard/vmContainerFactory.go

```diff
@@ -89,9 +94,28 @@ func (vmf *vmContainerFactory) Create() (process.VirtualMachinesContainer, error
 		return nil, err
 	}
 
+	// The vmContainerFactory keeps references to all containers it has created,
+	// in order to replace, from within the containers, those VM instances that
+	// become out-of-date after specific epochs.
+	vmf.containers = append(vmf.containers, container)
```

Comment:
Currently `createScQueryService()` from `main.go` creates these containers in a loop, one per `SCQueryService`. It doesn't look like `SCQueryService` instances can share a single container (and single Arwen instance) because they will execute queries concurrently.

Reply:
There is no need for this list. Every SCQueryService has a separate vmContainerFactory - which makes a separate set of vmContainers. Please remove the list.

Reply:
createScQueryElement -> is called for every query service and each of those makes one vmContainerFactory. Thus no need for list